makeflow/src/makeflow_gc.c

```diff
@@ -147,16 +147,14 @@ int makeflow_clean_file( struct dag *d, struct batch_queue *queue, struct dag_fi
 		makeflow_log_file_state_change(d, f, DAG_FILE_STATE_DELETE);
 
 		if(!silent)
-			debug(D_NOTICE, "Makeflow: Deleted path %s\n", f->filename);
+			debug(D_MAKEFLOW_RUN, "Makeflow: Deleted path %s\n", f->filename);
```

Comment:
Are you ok with makig it a fprintf(stdout, ...)? A D_NOTICE does not seem to fit here.